MinPlatformPkg/Test/Library/TestPointCheckLib/MmTestPointCheckLib.c

```diff
@@ -0,0 +1,134 @@
+/** @file
+
+Copyright (c) 2017 - 2018, Intel Corporation. All rights reserved.<BR>
+Copyright (c) 2022, Microsoft Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <PiMm.h>
+#include <Library/TestPointCheckLib.h>
+#include <Library/TestPointLib.h>
+#include <Library/BaseLib.h>
+#include <Library/DebugLib.h>
+#include <Library/BaseMemoryLib.h> 
+#include <Library/MemoryAllocationLib.h>
+#include <Library/MmServicesTableLib.h>
+#include <Guid/MemoryAttributesTable.h>
+
+#include "TestPointInternal.h"
+
+GLOBAL_REMOVE_IF_UNREFERENCED EFI_GUID  mTestPointSmmCommunciationGuid = TEST_POINT_SMM_COMMUNICATION_GUID;
+
+EFI_STATUS
+TestPointCheckSmrr (
+  VOID
+  );
+
+VOID
+TestPointDumpMmLoadedImage (
+  VOID
+  );
+
+EFI_STATUS
```

Comment:
Why are these function prototypes in this file?

Reply:
Removed some unnecessary prototypes.  Local function prototypes are still there.